build-ps/percona-server.spec

```diff
@@ -495,10 +500,15 @@ datadir=$(/usr/bin/my_print_defaults server mysqld | grep '^--datadir=' | sed -n
 %else
 /sbin/chkconfig --add mysql
 %endif
-# We need this because we don't provide my.cnf on centos 6
-# and the default system one doesn't have info needed
-%if 0%{?rhel} < 7
-if [ $1 -eq 1 ]; then
+if [ ! -f /etc/my.cnf ]
```

Comment:
Do we want to run this whole block on every minor upgrade?
Previously it was in "if [ $1 -eq 1 ]; then" which means the sed command from below and other stuff was only running during the package install and not on upgrade.